pkgs/top-level/all-packages.nix

```diff
@@ -14825,6 +14825,7 @@ in
 
   # Hardened linux
   hardenedLinuxPackagesFor = kernel: linuxPackagesFor (kernel.override {
+    features.ia32Emulation = false;
```

Comment:
I think we should keep the original naming of ```IA32_EMULATION``` to reduce need for additional documentation. Ideally we could just look directly. This is partially done in #42838.